assets/src/application/ui/input/BaseInput/style.scss

```diff
@@ -6,4 +6,71 @@
 		margin: var(--ee-margin-micro) var(--ee-margin-micro) 0;
 		white-space: nowrap;
 	}
+
+	.ant-input,
+	.ant-select .ant-select-selector,
+	.ant-select:not(.ant-select-disabled) .ant-select-selector,
+	.ant-select-single:not(.ant-select-customize-input) .ant-select-selector {
+		background-color: var(--ee-background-color);
+		background-clip: padding-box;
+		border: 2px solid var(--ee-color-grey-10);
+		border-radius: var(--ee-border-radius-small);
+		box-shadow: none;
+		box-sizing: border-box !important;
+		color: var(--ee-default-text-color);
+		display: inline-block;
+		font-size: var(--ee-font-size-default);
+		line-height: var(--ee-font-size-bigger);
+		height: var(--ee-icon-button-size) !important;
+		transition: border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
+		vertical-align: middle;
+		width: 100%;
+	}
+
+	.ant-input {
+		padding: var(--ee-padding-micro) var(--ee-padding-tiny);
+
+		&:hover,
+		&:focus {
+			border-color: unset;
+			border-right-width: 2px !important;
+			border: 2px solid var(--ee-color-primary-low-contrast);
+			box-shadow: none;
+			outline: none;
+		}
+	}
+
+	.ant-select,
+	.ant-select:not(.ant-select-disabled),
+	.ant-select-single:not(.ant-select-customize-input) {
+		.ant-select-selector {
+			padding: 0 var(--ee-padding-default) 0 var(--ee-padding-tiny);
+		}
+
+		&:hover,
+		&:focus,
+		&.ant-select-focused {
+			.ant-select-selector,
+			.ant-select-selector:focus,
+			.ant-select-selector:hover,
+			&:hover .ant-select-selector,
+			&:focus .ant-select-selector {
+				border-color: unset;
```

Comment:
quite readable nesting

Reply:
despite it being a little crazy because of how that component works